### ISSUE TITLE
Rocket and Sticky Jumping Is Too Powerful

### ISSUE THREAD

**Issue by Alexandroid99** [RaraFolf/TF2HLCoop#25]
In the current state, rocket and sticky jumps can sometimes be used to skip entire sections of a map. Of course the areas weren't made with these mechanics in mind, but more moderate jump physics would balance it out a little bit and reduce the need for map fixes.

**Comment by MerekiDor** [comment]
I guess in future versions it will be fixed. Actually if you simply add invisible walls on all the maps, then the problem is solved. But you know, there's 65~ maps and it will be really hard.

**Comment by kem008** [comment]
Ye it can easy cut other maps. Maps not designed for TF2 characters.

**Comment by Alexandroid99** [comment]
Invisible walls are not the solution as you can see in normal TF2 maps: ie a roof seems traversible, while you actually bump into an invisible wall and ontop of splash damage you take fall damage as well.

**Comment by MerekiDor** [comment]
Alexandroid99,
Then if it isnt a solution, then i don't know.

**Comment by xXxToxicBlueDustxXx** [comment]
You can make the Rocket/Sticky Jump deactivate trough a Cvar for example if
`lf_coop 1` is on deactivate some TF2 Mechanics That Don't fit in HL2 Maps is a cheap fix but should work

**Comment by Msalinas2877** [comment]
Or maybe just reduce the rocket/sticky jump impulse on players.

**Comment by xXxToxicBlueDustxXx** [comment]
But This Will Break The Gameplay of some TF2 Maps so is  easy to use a ConVar For This

**Comment by Msalinas2877** [comment]
You shouldnt be using this mod to play Normal TF2

**Comment by Alexandroid99** [comment]
Having a convar for sort of a rocket jump force multiplier would be a compromise, but I don't get why you would want to play standard TF2 maps on this...

**Comment by xXxToxicBlueDustxXx** [comment]
because if in a far future this mod add a SMOD's mapadd will be TF2's 2007 maps modified trough a simple .txt for example adding NPCs or Event Like a SMOD Redux or Obisidian Conflict whit the CSS maps

**Comment by Msalinas2877** [comment]
Yeah, and in a far future Half-Life 3 may release, or in a far future we the humans may have superpowers -_- , seriously just because you think about something it doesnt mean it will happen "in a far future", your statement is garbage

**Comment by xXxToxicBlueDustxXx** [comment]
@Msalinas2877 I'm just giving an opinion you do not need to use violence to respond because as the development goes many will say that this mod is dead in my opinion clear

**Comment by RaraFolf** [comment]
I will tell you personally that playing on live TF2 maps, even with the idea of using a SMod style mapadd, will never be in the cards.
So we would never balance mechanics out around it.

**Comment by xXxToxicBlueDustxXx** [comment]
but SMOD Style Map add could fix or add someone bugs ( it can't replace engine based func_ ) i'm referring for the TF2 2007 maps not the TF2 Live maps it can be a cheap way because Decompiling and recompiling HL2 its a hard task

**Comment by ISNEWPuddy** [comment]
https://valvearchive.btr.pm/   had all hl2 offical .vmf maps execpt its just closed by valve

**Comment by xXxToxicBlueDustxXx** [comment]
i known this site is the best for HL2 Modders because you can found HL2's Full mapsrc but its shutdown the only way i kwon its decompiling these maps but can break them so the cheap way is using Mapadd whit mapadd you can create labels that can execute a action

**Comment by Alexandroid99** [comment]
May I just interject the fact that I opened this issue to comment on rocket jumping alone...

**Comment by xXxToxicBlueDustxXx** [comment]
ok, so this is the general problem:

1. Make The Rocket/Sticky Jump Lees Powerful without removing it ( because its the main mechanic of this )

2. Recompiling HL2 maps for The TF2's Gameplay is to hard (implying that's over 62 maps)

3.  Make This Mechanic on/off through a Cvar ( for example `lf_jumping` ) without removing it
